content/en/apps/tutorials/death-reporting.md

```diff
@@ -0,0 +1,101 @@
+
+- View the death report menu
+- Open, fill and submit a death report form for a contact
+- Display all the deceased contacts of an area
+
+## Required resources
+
+Before you begin you need to :
+
+1. have a working CHT development environment (see docs or this)
+2. Know how to build an app form in the CHT ([https://docs.communityhealthtoolkit.org/apps/tutorials/app-forms/](https://docs.communityhealthtoolkit.org/apps/tutorials/app-forms/))
+3. know how to set form properties ([https://docs.communityhealthtoolkit.org/apps/tutorials/form-properties/](https://docs.communityhealthtoolkit.org/apps/tutorials/form-properties/))
+4. know how to configure tasks ([https://docs.communityhealthtoolkit.org/apps/tutorials/tasks/](https://docs.communityhealthtoolkit.org/apps/tutorials/tasks/))
+5. know about targets ([https://docs.communityhealthtoolkit.org/apps/tutorials/targets/](https://docs.communityhealthtoolkit.org/apps/tutorials/targets/))
+
+We will briefly explain most of these concepts on the fly, but it is important to understand them well in order to get the most out of this guide.
+
+## Implementation Steps
+
+### 1. Layout the form
+
+To create the form view, the first step is to create the XLSX file that best matches the proposed design. It should be noted that a death report form is not very long (usually two pages, one for filling in the elements of the report and another for the summary). An example death form XLSX file below:
+
+| name | label |
+| ---- | ----- |
+|      |       |
+
+Then you have to compile the form through the medic-conf tool to convert it into a view in the application.
+The form compilation phase then generates two additional files. An <form_code>.xml file which actually corresponds to the file loaded and interpreted by the CHT for the view and another <form_code> .properties.json file. <form_code> being the form identifier (very often the identifier is also the name of the lowercase form with spaces replaced by underscores). We will not touch the XML file, it is just a file generated directly from the XLSX file. The file that interests us for the rest is the <form_code> .properties.json.
+
+### 2. Edit the form properties.json file
+
+The <form_name>.properties.json file allows you to add logic that ensures that the right action appears for the right contacts (people and places). For instance, a death form will only appear for person contacts on the CHT who are alive. You can learn more about form properties here ([https://docs.communityhealthtoolkit.org/apps/tutorials/form-properties/](https://docs.communityhealthtoolkit.org/apps/tutorials/form-properties/))
+In our case, if the file doesn’t already exist, you’ll create a file named <form_name>.properties.json in the forms/app folder. Then open that file and add the following code.
+
+```json
+{
+	"context":  {
+		"expression":  "contact.type === 'person' && summary.alive && !summary.muted && user.parent.type === 'health_center'",
```

Comment:
don't you need to update contact-summary? should that be first?

Reply:
Yeah! but I was thinking of having a whole section explaining everything that happens in the contact-summary. And then there I'll mention the link with the context in the .properties.json file. What do you think?